.gitmodules

```diff
@@ -18,7 +18,7 @@
 	url = https://github.com/alire-project/simple_logging.git
 [submodule "deps/ada-toml"]
 	path = deps/ada-toml
-	url = https://github.com/pmderodat/ada-toml.git
+	url = https://github.com/mosteo/ada-toml.git
```

Comment:
It’s not clear to me why we need this, but if it’s important to have a separate fork, maybe hosting this fork in alire-project would be better, no?

Reply:
It's until my two PRs to your repo are merged, at which point I plan to switch back to yours. Is there a better way to do this?

I thought about forking under alire, but since I didn't intend for it to become the main one I thought it could cause confusion.

Reply:
Understood. I think the best way would be to close the loop on your PR for ada-toml :-) This shouldn’t take long: the first one is already merged: do you need help for what’s missing? (only the testcase IIRC).